src/openms/source/DATASTRUCTURES/QTCluster.cpp

```diff
@@ -286,11 +306,14 @@ namespace OpenMS
     quality_ = (max_distance_ - internal_distance) / max_distance_;
   }
 
-  OpenMSBoost::unordered_map<Size, std::vector<GridFeature*> > QTCluster::getAllNeighbors() 
+  OpenMSBoost::unordered_map<Size, std::vector<GridFeature*> > QTCluster::getAllNeighbors() const
   {
     OPENMS_PRECONDITION(finalized_,
         "Cannot perform operation on cluster that is not finalized")
 
+    // get references on member that is used in this function
+    NeighborMap& neighbors_ = data_->neighbors_;
```

Comment:
not needed. Jut go range-based for...

Reply:
Done. The range-based for loops also really helped to improve the readability.